src/Language/PureScript/Docs/RenderedCode/Types.hs

```diff
@@ -159,7 +161,10 @@ syntax :: String -> RenderedCode
 syntax x = RC [Syntax x]
 
 ident :: String -> RenderedCode
-ident x = RC [Ident x]
+ident x = RC [Ident x ThisModule]
+
+ident' :: String -> ContainingModule -> RenderedCode
+ident' x m = RC [Ident x m]
```

Comment:
@hdgarrood I didn't find the `ContainingModule` really being used anywhere in the code we have here, but I assume pursuit uses it to generate links between modules in the documentation? 

`Ident` didn't previously need this (we'd only show idents for the current module, but now an external ident can appear in the guise of a type operator). So I guess this will also be a breaking change for Pursuit.